app/pages/admin/project-status.jsx

```diff
@@ -0,0 +1,160 @@
+import React, { Component } from 'react';
+import apiClient from 'panoptes-client/lib/api-client';
+
+import ProjectIcon from '../../components/project-icon';
+import getWorkflowsInOrder from '../../lib/get-workflows-in-order';
+import WorkflowToggle from '../../components/workflow-toggle';
+import LoadingIndicator from '../../components/loading-indicator';
+
+import VersionList from './project-status/version-list';
+import ExperimentalFeatures from './project-status/experimental-features';
+import Toggle from './project-status/toggle';
+import RedirectToggle from './project-status/redirect-toggle';
+
+class ProjectStatus extends Component {
+  constructor(props) {
+    super(props);
+    this.onChangeWorkflowLevel = this.onChangeWorkflowLevel.bind(this);
+    this.getWorkflows = this.getWorkflows.bind(this);
+    this.rerender = this.rerender.bind(this);
+    this.renderError = this.renderError.bind(this);
+    this.renderWorkflows = this.renderWorkflows.bind(this);
+
+    this.state = {
+      project: null,
+      error: null,
+      usedWorkflowLevels: [],
+      workflows: []
+    };
+  }
+
+  componentDidMount() {
+    this.getProject().then(() => this.getWorkflows());
+  }
+
+  componentWillUnmount() {
+    this.state.project.stopListening('change', this.rerender);
+  }
+
+  onChangeWorkflowLevel(workflow, event) {
+    this.setState({ error: null });
+    let selected = event.target.value;
+    selected = selected === 'none' ? undefined : selected;
+    return workflow.update({ 'configuration.level': selected }).save()
+      .then(() => this.getWorkflows())
+      .catch(error => this.setState({ error }));
+  }
+
+  rerender() {
+    if (this.isMounted()) {
```

Comment:
The latest version of React is unhappy with us using this: 
```
Warning: isMounted(...) is deprecated in plain JavaScript React classes. Instead, make sure to clean up subscriptions and pending requests in componentWillUnmount to prevent memory leaks.
Uncaught TypeError: this.isMounted is not a function(…)
```